packages/client/components/promptResponse/loomExtension.ts

```diff
@@ -0,0 +1,89 @@
+import {Node} from '@tiptap/core'
+import {JSONContent, mergeAttributes} from '@tiptap/react'
+
+const LOOM_REGEX = /^(?:https:\/\/)?(?:www\.)?loom\.com\/share\/[a-zA-Z0-9]*\?.*$/g
```

Comment:
-1 i don't know how i managed to get an insecure http link while copying & pasting across task cards, but somehow i lost the s & it didn't unfurl. can we use `https?` instead?

Alternatively, I found the loom official regex, which includes their staging URLs 🤣 . Could use that instead https://www.unpkg.com/browse/@loomhq/loom-embed@1.2.4/dist/esm/common.js

Reply:
Ooo yes, nice catch!

Including the staging URLs doesn't feel quite right to me 😂 but I made some minor modifications to our regex based on this, so nice find!